services/dynamic-sidecar/src/simcore_service_dynamic_sidecar/modules/long_running_tasks.py

```diff
@@ -315,8 +315,9 @@ async def _send_resource_tracking_stop(platform_status: SimcorePlatformStatus):
         # NOTE: https://github.com/ITISFoundation/osparc-simcore/issues/4952
         await _send_resource_tracking_stop(SimcorePlatformStatus.OK)
         raise
-
-    await ensure_read_permissions_on_user_service_data(mounted_volumes)
+    finally:
+        with log_context(_logger, logging.INFO, "ensure read permissions"):
+            await ensure_read_permissions_on_user_service_data(mounted_volumes)
```

Comment:
TIP:  make sure it does not raise!

Reply:
it is ok if it raises here. something is wrong